infrastructure/modules/s3/variables.tf

```diff
@@ -13,7 +13,7 @@ variable "lifecycle_rule_inputs" { default = [] }
 variable "force_destroy" {
   description = "Whether to forcefully destroy the bucket when it contains objects"
   type        = bool
-  default     = false
+  default     = true
```

Comment:
Is this part of this PR or is it already in main ? If already in main would not expect to see it listed as a change in. a branch rebased from main

Reply:
changed back to false